charts/pgadmin4/values.yaml

```diff
@@ -157,6 +237,11 @@ ingress:
 # Additional config maps to be mounted inside a container
 # Can be used to map config maps for sidecar as well
 extraConfigmapMounts: []
+  # - name: config-local
```

Comment:
So from my understand, if we enable config_local this becomes a requirement. Perhaps we should make an if statement within deployments.yaml and automatically add this configmap / secrets without configuring it with extraConfig, if this make sense? What are your thoughts?